example/unicorn/components/nested/actions.py

```diff
@@ -0,0 +1,21 @@
+from django_unicorn.components import UnicornView
+from example.coffee.models import Flavor
+
+
+class ActionsView(UnicornView):
+    model: Flavor = None
+    is_editing = False
+
+    def edit(self):
+        self.is_editing = True
+
+        # this doesn't do what you expect because resulting dom is scoped to
+        # this component and the parent component won't get morphed
+        self.parent.is_editing = True
```

Comment:
@joshiggins So, I tried to add an example and kind of stumbled at this point. Your code "works" so I could merge it in as is, but `Unicorn` doesn't handle child components in a very intuitive way (I don't think). I'd love to hear any ideas about how this might work better.

Reply:
I've put some ideas in #88, makes sense to hold this PR off for a bit since it really only solves naive use case where components are nested but otherwise independent.